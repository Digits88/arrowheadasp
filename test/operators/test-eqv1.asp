Test logical EQV operator:
<hr>

True eqv True =         <%= True eqv True %><br>
True eqv False =        <%= True eqv False %><br>
IsNull(True eqv Null) =	<%= IsNull(True eqv Null) %><br>
False eqv True =        <%= False eqv True %><br>
False eqv False =       <%= False eqv False %><br>
IsNull(False eqv Null) =<%= IsNull(False eqv Null) %><br>
IsNull(Null eqv True) = <%= IsNull(Null eqv True) %><br>
IsNull(Null eqv False) =<%= IsNull(Null eqv False) %><br>
IsNull(Null eqv Null) = <%= IsNull(Null eqv Null) %><br>

