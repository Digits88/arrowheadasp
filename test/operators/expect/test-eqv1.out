Test logical EQV operator:
<hr>

True eqv True =         True<br>
True eqv False =        False<br>
IsNull(True eqv Null) =	True<br>
False eqv True =        False<br>
False eqv False =       True<br>
IsNull(False eqv Null) =True<br>
IsNull(Null eqv True) = True<br>
IsNull(Null eqv False) =True<br>
IsNull(Null eqv Null) = True<br>

